src/components/home/logged-out.jsx

```diff
@@ -30,7 +29,25 @@ export default class LoggedOutHome extends React.PureComponent {
       <div>
         {this.renderTitleBar()}
         <div className="body-with-title-bar container">
-          logged out wew
+        <div className="description">
+          <div className="header1">
+          the alumni-powered
+          </div>
+          <div className="header2">scholarship</div>
+          <div> built by </div>
+          <div> fullstack academy alumni for future alumni
+          </div>
+        </div>
+          <div className="row">
```

Comment:
for readability can you break this out to it's own submethod? Something like `renderImages`

Same with the header portion